matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/EventRelationsAggregationTask.kt

```diff
@@ -42,6 +44,14 @@ internal interface EventRelationsAggregationTask : Task<EventRelationsAggregatio
     )
 }
 
+enum class VerificationState {
```

Comment:
could be `private`

Reply:
It's part of API, used to display in timeline